src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.domain.Fruit;
+
+public interface FruitDao {
+    void add(Fruit fruit);
```

Comment:
The `add` method should likely have a return type to indicate the success of the operation, or it should accept additional parameters if needed to perform the operation. Moreover, consider defining more methods to handle other CRUD operations for better DAO interface design.